app/src/lib/tokens/PopupMenu.svelte

```diff
@@ -1,6 +1,9 @@
+<script>
+	export let color_classes = 'bg-info text-info-content'
```

Comment:
I'm wondering if this is premature... I only see it being used in one place at this time.

Reply:
`Result` and `NotFound` use `PopupMenu` and use the default info colors. The new `Message` needed to use different colors. I'm open to other suggestions though!